definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,58 @@
+## Various latitude/longitude properties
+
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
+
+These values are machine-generated math centroids for a record's geometry.
+
+### intersection
+
+Find out more about the `intersection` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/intersection).
+
+The `intersection:latitude` and `intersection:longitude` represent the intersection location for a given record.
```

Comment:
Nearest to the record, centrally located to the record? What's the editing guidance here?

Reply:
Is the intended use navigation? Does it reference the lat lng of the intersection placetype polygon?

/cc @thisisaaronland

Reply:
See @thisisaaronland answer in https://github.com/whosonfirst/whosonfirst-cookbook/pull/17#discussion_r151817029.

I'm confused: Isn't the intersection ID the hierarchy in that case? Are we copying the intersection records geom lat/lng into any random feature's intersection properties then? What if it changes?

Reply:
Clarifying here that the `intersection:*` values are the lat/long values used in the initial process to create voronoi polygons. I've updated the doc to reflect this.